src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2921,6 +2921,11 @@ void processContextElement
         buildGeneralErrorResponse(ceP, NULL, responseP, SccInvalidModification, "Already Exists");
         return;
     }
+    else if ((checkEntityExistance == true) && (action == "APPEND") && (entitiesNumber == 0))
```

Comment:
Matter of opinion. I prefer to have the '== true' there.
If you want to, we decide a way to do this and we add it to the style guide.
I don't have an especially strong opinion.
I prefer adding the == true for clarity but I know most people dislike it.